app/src/main/java/com/example/cs65_final_project/Ingredient.java

```diff
@@ -17,6 +17,11 @@ public Ingredient(String name, float amount, @Nullable String aisle) {
         this.aisle = aisleClassifier(aisle);
     }
 
+    public Ingredient(String name, String aisle){
```

Comment:
Should we just remove `float amount` from the above constructor?

Reply:
Same thing here, just an optional constructor,